ast/builtins.go

```diff
@@ -783,6 +786,17 @@ var Sprintf = &Builtin{
 	),
 }
 
+// ParseNumBytes converts strings like 10GB, 5K, 4mb, and the like into an integer number of bytes.
+var ParseNumBytes = &Builtin{
+	Name: "parse_num_bytes",
```

Comment:
(duplicating recent comment on old thread here).

Let's call this `units.parse_bytes` unless there are major objections.

Reply:
Sounds good to me